.github/workflows/mongodb-oplog-workflow.yml

```diff
@@ -8,14 +8,11 @@ on:
     paths:
       - 'support/checkstyle/**'
       - 'support/ide-configs/**'
```

Comment:
This one shouldn't be needed either? If you agree, can you rework the commit, force-push, and merge this PR? Thx!

Reply:
Agree that the `ide-configs` can be removed. The `checkstyle` will be needed though.